[2025-01-14T00:00:09.056+0000] {processor.py:161} INFO - Started process (PID=10568) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:00:09.056+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:00:09.057+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:00:09.057+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:00:09.236+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:00:09.235+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:00:09.237+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:00:09.251+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.198 seconds
[2025-01-14T00:00:32.970+0000] {processor.py:161} INFO - Started process (PID=10594) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:00:32.970+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:00:32.971+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:00:32.971+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:00:33.190+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:00:33.189+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:00:33.191+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:00:33.207+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.239 seconds
[2025-01-14T00:01:04.001+0000] {processor.py:161} INFO - Started process (PID=10653) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:01:04.002+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:01:04.003+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:01:04.002+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:01:04.176+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:01:04.175+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:01:04.176+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:01:04.189+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.190 seconds
[2025-01-14T00:01:34.331+0000] {processor.py:161} INFO - Started process (PID=10712) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:01:34.332+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:01:34.332+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:01:34.332+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:01:34.494+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:01:34.493+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:01:34.494+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:01:34.508+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.180 seconds
[2025-01-14T00:02:05.136+0000] {processor.py:161} INFO - Started process (PID=10771) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:02:05.137+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:02:05.138+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:02:05.138+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:02:05.297+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:02:05.296+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:02:05.297+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:02:05.310+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.176 seconds
[2025-01-14T00:02:36.008+0000] {processor.py:161} INFO - Started process (PID=10830) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:02:36.009+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:02:36.010+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:02:36.010+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:02:36.171+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:02:36.170+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:02:36.171+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:02:36.184+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.178 seconds
[2025-01-14T00:03:06.402+0000] {processor.py:161} INFO - Started process (PID=10889) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:03:06.403+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:03:06.403+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:03:06.403+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:03:06.565+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:03:06.564+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:03:06.565+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:03:06.579+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.179 seconds
[2025-01-14T00:03:37.369+0000] {processor.py:161} INFO - Started process (PID=10948) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:03:37.370+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:03:37.371+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:03:37.370+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:03:37.534+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:03:37.533+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:03:37.534+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:03:37.546+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.179 seconds
[2025-01-14T00:04:08.395+0000] {processor.py:161} INFO - Started process (PID=11007) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:04:08.395+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:04:08.396+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:04:08.396+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:04:08.566+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:04:08.565+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:04:08.566+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:04:08.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.186 seconds
[2025-01-14T00:04:39.538+0000] {processor.py:161} INFO - Started process (PID=11068) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:04:39.539+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:04:39.540+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:04:39.539+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:04:39.721+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:04:39.720+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:04:39.721+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:04:39.734+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.198 seconds
[2025-01-14T00:05:09.809+0000] {processor.py:161} INFO - Started process (PID=11127) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:05:09.810+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:05:09.810+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:05:09.810+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:05:09.990+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:05:09.989+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:05:09.990+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:05:10.003+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.197 seconds
[2025-01-14T00:05:25.661+0000] {processor.py:161} INFO - Started process (PID=11178) to work on /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:05:25.662+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/weather_pipeline_dag copy.py for tasks to queue
[2025-01-14T00:05:25.662+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:05:25.662+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:05:25.829+0000] {logging_mixin.py:188} INFO - [2025-01-14T00:05:25.828+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/weather_pipeline_dag copy.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_pipeline_dag copy.py", line 8, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-01-14T00:05:25.830+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_pipeline_dag copy.py
[2025-01-14T00:05:25.845+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/weather_pipeline_dag copy.py took 0.186 seconds
